db/Repository.go

```diff
@@ -44,6 +45,10 @@ func (r Repository) GetDirName(templateID int) string {
 	return r.getDirNamePrefix() + "template_" + strconv.Itoa(templateID)
 }
 
+func (r Repository) GetAnsiblePath(templateID int) string {
```

Comment:
I don't think `GetAnsiblePath` is a correct name, because it can be Terraform or any other repository.